apps/server/src/infra/sync/tsp/tsp-legacy-migration.service.integration.spec.ts

```diff
@@ -1,15 +1,15 @@
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { MongoMemoryDatabaseModule } from '@infra/database';
 import { EntityManager } from '@mikro-orm/mongodb';
+import { SystemType } from '@modules/system';
```

Comment:
Die Imports widersprechen der eslint-Regel "infra-modules may NOT import from apps, @core, @modules, or migrations". Ist das gewollt?